src/lib/components/chips/Chips.component.js

```diff
@@ -1,144 +1,165 @@
 //@flow
 import * as defaultTheme from "../../style/theme";
-import type { Variant } from "../constants";
+import type { Variant, Size } from "../constants";
 import type { Node } from "react";
 import styled, { css } from "styled-components";
-
+import Button from "../button/Button.component";
 import React from "react";
 import { lighten } from "polished";
 import { mergeTheme } from "../../utils";
 
 type Props = {
   text: string,
-  color?: string,
   variant?: Variant,
   icon?: Node,
-  buttonIcon?: Node,
   onClick?: any => void,
-  onRemove?: any => void
+  onRemove?: any => void,
+  size?: Size
 };
 
 const ChipsContainer = styled.div`
   display: inline-flex;
   background-color: ${defaultTheme.grayLighter};
-  border-radius: 15px;
+  .sc-chips-remove {
+    padding-right: 10px;
+    color: ${defaultTheme.white};
+    &:hover {
+      color: ${defaultTheme.gray};
```

Comment:
We should not use ${defaultTheme.gray}. If it's a gray Chip, the `remove icon` disappears when you hover.
I will let you address it after merging the dark/light mode PR.